### PR TITLE
Added Japanese Pixel Styling for Login, Signup & HomePage

### PR DESCRIPTION
**Pull request by forreya**

### CODE REVIEW

**Review by edwardfgt — APPROVED** [review]
Looks great.

File: frontend/src/components/signup/Signup.component.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ const Signup = () => {
 
   return (
     <>
-      <div className="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
+      {/* <div className="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
```
Comment (SimpleLuke) [inline frontend/src/components/signup/Signup.component.js:86]: You may remove unused code.

File: frontend/src/components/signup/Signup.component.js [hunk 2]
```diff
@@ -158,6 +158,69 @@ const Signup = () => {
...
+
+      <div className="min-h-full min-w-full bg-pixelSepia bg-cover pixel-font">
+        <form
+          className=""
+          onSubmit={handleSubmit}
+          data-test="SignupForm"
+        >
+          <div className="h-screen flex justify-center items-center">
+            <div className="m-7 flex flex-col relative w-2/5">
+              <div className="bg-japanese-brown-2 mt-7 shadow-2xl sm:rounded-xl">
```
Comment (SimpleLuke) [inline frontend/src/components/signup/Signup.component.js:171]: Is this brown box under the form needed?

File: frontend/src/components/login/Login.component.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ const Login = () => {
 
   return (
     <>
-      <div className="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
+      {/* <div className="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
```
Comment (SimpleLuke) [inline frontend/src/components/login/Login.component.js:47]: You may clear up unused comment code.

File: frontend/src/components/login/Login.component.js [hunk 4]
```diff
@@ -125,6 +125,70 @@ const Login = () => {
...
+          <div className="h-screen flex justify-center items-center">
+            <div className="m-7 flex flex-col relative w-2/5">
+              <div className="bg-japanese-brown mt-7 shadow-2xl sm:rounded-xl">
+                <div className="p-5 text-white">
+                  <h1 className="text-8xl md:text-5xl lg:text-7xl">.</h1>
+                  <p className="text-base md:text-lg lg:text-8xl">.</p>
+                  <p className="sm:mt-32 text-base md:text-lg lg:text-7xl">.</p>
+                </div>
+              </div>
+              <input type="hidden" name="remember" defaultValue="true" />
```
Comment (SimpleLuke) [inline frontend/src/components/login/Login.component.js:146]: What is this input for?

File: frontend/src/components/signup/Signup.component.js [hunk 5]
```diff
@@ -158,6 +158,69 @@ const Signup = () => {
...
+                <div className="p-5 text-white">
+                  <h1 className="text-8xl md:text-5xl lg:text-7xl">.</h1>
+                  <p className="text-base md:text-lg lg:text-8xl">.</p>
+                  <p className="sm:mt-32 text-base md:text-lg lg:text-7xl">.</p>
+                </div>
+              </div>
+              <input type="hidden" name="remember" defaultValue="true" />
```
Comment (SimpleLuke) [inline frontend/src/components/signup/Signup.component.js:178]: What is this input for?

File: frontend/src/components/signup/Signup.component.js [hunk 6]
```diff
@@ -158,6 +158,69 @@ const Signup = () => {
...
+        <form
+          className=""
+          onSubmit={handleSubmit}
+          data-test="SignupForm"
+        >
+          <div className="h-screen flex justify-center items-center">
+            <div className="m-7 flex flex-col relative w-2/5">
+              <div className="bg-japanese-brown-2 mt-7 shadow-2xl sm:rounded-xl">
+                <div className="p-5 text-white">
+                  <h1 className="text-8xl md:text-5xl lg:text-7xl">.</h1>
```
Comment (SimpleLuke) [inline frontend/src/components/signup/Signup.component.js:173]: If this is a design element, maybe it is better not to use h1 tag for a dot?

**Review by SimpleLuke — CHANGES_REQUESTED** [review]

File: frontend/src/components/login/Login.component.js (hunk 4 above)
Comment (forreya) [reply frontend/src/components/login/Login.component.js:146]: it was from the previous code design, I didn't know what it did so I copied it over into the new design.

File: frontend/src/components/signup/Signup.component.js (hunk 6 above)
Comment (forreya) [reply frontend/src/components/signup/Signup.component.js:173]: i think ill leave it for now and change it in the future

**Review by annamaggg — APPROVED** [review]
bbeeeaauuuttiiiifffullll

**Review by forreya — COMMENTED** [review]
Anything else?

**Review by forreya — COMMENTED** [review]
Made all requested changes

**Review by SimpleLuke — APPROVED** [review]

**Pull request merged**